doc/whats_new/v0.24.rst

```diff
@@ -256,6 +256,18 @@ Changelog
   :pr:`18280` by :user:`Alex Liang <tianchuliang>` and
   `Guillaume Lemaitre`_.
 
+- |Enhancement| :func:`datasets.fetch_openml` now supports categories with
```

Comment:
We need to move these entries in `v1.0.rst`.

Reply:
done